oct_converter/readers/binary_structs/fds_binary.py

```diff
@@ -23,3 +23,11 @@
     "unknown" / PaddedString(1, "ascii"),
     "size" / Int32un,
 )
+
+# ref: https://github.com/neurodial/LibOctData/blob/master/octdata/import/topcon/topconread.cpp#L318
+scan_param_04 = Struct(
+    "unknown" / Array(3, Int32un),
+    "x_width_mm" / Float64n,
+    "y_width_mm" / Float64n,
+    "z_resolution_um" / Float64n
+)

```

Comment:
Could you rename these to match those in the `.fda` binary? I'm going to try to bring the `.fds` reader up to parity with the `.fda` soon and it will be easier if they stick to the same naming. If you think these names are better than the in .fda happy to have the .fda versions updated to match these!

Reply:
Sure! Do you mean just the struct name or also the individual params in the struct?  And if so, how should I decide on the names, I don't think the ones from .fda (`tomogram_{x}_dimension_in_mm`) necessarily are the same here.

Reply:
Hm i meant everything. I just checked for the fda/fds files from the biobank and they both give exactly the same values  so I assume they do refer to the same thing. I think I just took the metadata spec from a different source which gave them slightly different names

Reply:
Sounds good, based on [this](https://bitbucket.org/uocte/uocte/wiki/Topcon%20File%20Format), I think I'm going to rename both fds and fda to 
```python
x_size_mm
y_size_mm
z_resolution_um
```

thoughts?

Reply:
Sounds good to me!